### DISCUSSION TITLE
Users

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6170] [Q&A]
I want to modify the users on add new user or update existing one to add new field : (phone) under email field and i want to change the code to validate the phone to be unique and only 11 numbers. Please need this feature ASAP

**Comment by duncanmcclean** [comment]
Add a phone field to your Users blueprint and add a validation rule for `max:11`?

**Reply by HazemMohamed252** [reply]
What about to make it UNIQUE please ? Another thing this will show it on editing but on create new user no phone field will be shown. Any help ?

**Reply by jasonvarga** [reply]
Add `unique_user_value:{id}` as a rule.

https://statamic.dev/validation#unique-user-value
